frontend/src/components/admin/RoundViewCard.vue

```diff
@@ -44,27 +43,67 @@ it displays the information of one round so that a list of round views can easil
 </template>
 
 <script>
+import {RequestHandler} from "@/api/RequestHandler";
+import PlanningService from "@/api/services/PlanningService";
+import router from "@/router";
+
 export default {
   name: "RoundViewCard",
   data: () => ({
-    percentage: 100,
-    users: []
+    percentage: 0,
+    users: [],
+    count: 0
   }),
-  computed() {
-    this.percentage = this.data.round.status.filter(s => s === 'FI').length / this.data.round.ronde.buildings.length * 100;
+  created() {
+    this.getStatus();
   },
   props: {
     data: {
       type: Object,
       default: () => ({
-        round: {ronde: {name: ''}},
+        round: {ronde: {name: ''}, id: -1},
         date: ''
       })
     }
   },
+  watch: {
+    data: function() {
+      this.count++;
+      if (this.count % 2 === 0) this.getStatus();
+    }
+  },
   methods: {
+    router() {
+      return router
+    },
     info() {
-      // TODO
+      router.push({name: 'adminRoundView', query: {planning: this.data.round.id, date: this.data.date}});
+    },
+    getWeek(d) {
```

Comment:
Deze functie misschien in een aparte file zetten zodat hij niet telkens opnieuw gemaakt wordt en dan nog aanpassen dat de week op zondag begint. Maar dit kan eventueel in een andere pr gedaan worden.